src/components/table/OcTable.vue

```diff
@@ -249,15 +249,16 @@ export default {
       if (!this.dragDrop) return
       this.$emit(EVENT_FILE_DRAGGED, file)
     },
-    dropRowEvent(event) {
+    dropRowEvent(id, event) {
       if (!this.dragDrop) return
       const dropTarget = event.target
       const dropTargetTr = dropTarget.closest("tr")
       const dropFileId = dropTargetTr.dataset.fileId
+      this.dropRowStyling(id, true, event)
       this.$emit(EVENT_FILE_DROPPED, dropFileId)
     },
     dropRowStyling(id, leaving, event) {
-      if (event.currentTarget.contains(event.relatedTarget)) {
+      if (event.currentTarget !== null && event.currentTarget.contains(event.relatedTarget)) {
```

Comment:
Since we're using node >= 14 you could do this with optional chaining:
```suggestion
      if (event.currentTarget?.contains(event.relatedTarget)) {
```

Reply:
Good hint, thanks!